security/badware/index.html

```diff
@@ -40,6 +40,7 @@ <h2>Redirects</h2>
 
     <h2>Edge Cases</h2>
     <ul>
+        <li><a href="./phishing badͮ.html">Phishing Page With Abnormal URL Escape Sequences</a></li>
```

Comment:
I see that cheeky "dͮ". Is that the only trick here? Or is that space a part of it?